igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCacheV2KeysMigration.java

```diff
@@ -62,7 +63,7 @@
     private final AtomicBoolean running = new AtomicBoolean();
 
     @Autowired
-    public DockerRegistryCacheV2KeysMigration(RedisClientDelegate redis,
+    public DockerRegistryCacheV2KeysMigration(@Qualifier("redisClientDelegate") RedisClientDelegate redis,
```

Comment:
Why is this qualifier necessary?

Reply:
ha! had a case of two beans found and only one is needed failure when I first ran this. Verified later on that I don't need the qualifier. Just forgot to yank it out. Good catch sir!